CefSharp.Wpf/Internals/MouseAdjustor.cs

```diff
@@ -0,0 +1,132 @@
+using CefSharp.Structs;
+
+namespace CefSharp.Wpf.Internals
+{
+    public class MouseAdjustor
+    {
+        /// <summary>
+        /// The x-offset.
+        /// </summary>
+        private int xOffset;
+
+        /// <summary>
+        /// The y-offset.
+        /// </summary>
+        private int yOffset;
+
+        /// <summary>
+        /// The original-rect.
+        /// </summary>
+        private Rect originalRect;
+
+        /// <summary>
+        /// The teleporting-rect.
+        /// </summary>
+        private Rect teleportingRect;
+
+        public System.Windows.Point Update(Rect originalRect, Rect viewRect)
```

Comment:
Method needs xml comment and probably to be renamed.

Maybe `UpdatePopupSizeAndPosition`?

Reply:
Done.